README.md

```diff
@@ -1 +1,51 @@
-# jv-dao-service
\ No newline at end of file
+# HW 03
+- Establish connection to Database.
```

Comment:
Let's add business requirements for this HW. Or what problem we should solve in this HW